src/components/web-form-renderer.vue

```diff
@@ -12,9 +12,14 @@ except according to the terms contained in the LICENSE file.
 
 <template>
   <loading :state="initiallyLoading"/>
-  <template v-if="formVersionXml.dataExists">
-    <!-- update instanceId to rerender the component -->
-    <OdkWebForm :key="instanceId" :form-xml="formVersionXml.data" :fetch-form-attachment="getAttachment" @submit="handleSubmit"/>
+  <template v-if="dataExists">
+    <!-- update newInstanceId to rerender the component -->
+    <OdkWebForm
+      :key="newInstanceId"
```

Comment:
`:key` is removed in the upcoming PR https://github.com/getodk/central-frontend/pull/1178